packages/components/AssetsPagination.tsx

```diff
@@ -0,0 +1,177 @@
+              },
+            ]}
+          >
+            <TextInput
+              value={currentPage.toString()}
+              inputMode="numeric"
+              style={[
+                fontSemibold14,
+                {
+                  color: "white",
+                  flex: 1,
+                  width: 56,
+                },
+                { outlineStyle: "none" } as any,
+              ]}
+              onChangeText={(text) => {
+                handleChangePage(Number(text));
+              }}
+            />
+          </TertiaryBox>
+        </View>
+      </View>
+      <SpacerRow size={1} />
+      <View style={{ flex: 1 }}>
+        <View style={styles.section}>
+          <TouchableOpacity onPress={() => handleChangePage(0)}>
+            <LegacyTertiaryBox height={42} width={56}>
+              <SVG source={chevronLeftDoubleSVG} height={16} width={16} />
+            </LegacyTertiaryBox>
+          </TouchableOpacity>
+          <SpacerRow size={1} />
+          <TouchableOpacity onPress={() => handleChangePage(currentPage - 1)}>
+            <LegacyTertiaryBox height={42} width={56}>
+              <SVG source={chevronLeftSVG} height={16} width={16} />
+            </LegacyTertiaryBox>
+          </TouchableOpacity>
+          <SpacerRow size={2} />
+
+          <View style={[styles.section, { justifyContent: "flex-end" }]}>
+            <Box
+              notched
+              style={{
+                height: 42,
+                width: 80,
+                borderWidth: 1,
+                justifyContent: "center",
+                alignItems: "center",
+                backgroundColor: primaryColor,
+                borderColor: neutral33,
+              }}
+            >
+              <TouchableOpacity
+                style={{
+                  flexDirection: "row",
+                  alignItems: "center",
+                }}
+                activeOpacity={1}
+                disabled
+              >
+                <BrandText
+                  style={[fontSemibold14, { marginRight: layout.spacing_x1 }]}
+                >
+                  {maxPage}
+                </BrandText>
+              </TouchableOpacity>
+            </Box>
+          </View>
+          <SpacerRow size={2} />
+
+          <TouchableOpacity onPress={() => handleChangePage(currentPage + 1)}>
+            <LegacyTertiaryBox height={42} width={56}>
+              <SVG source={chevronRightSVG} height={16} width={16} />
+            </LegacyTertiaryBox>
+          </TouchableOpacity>
+          <SpacerRow size={1} />
+          <TouchableOpacity onPress={() => handleChangePage(maxPage - 1)}>
+            <LegacyTertiaryBox height={42} width={56}>
+              <SVG source={chevronRightDoubleSVG} height={16} width={16} />
+            </LegacyTertiaryBox>
+          </TouchableOpacity>
+        </View>
+      </View>
+    </View>
+  );
+};
+
+// FIXME: remove StyleSheet.create
+// eslint-disable-next-line no-restricted-syntax
```

Comment:
In fact, when we was refactoring the code, we decided to not use StyleSheet create anymore so we put the FIXME to fix when working on each component. We can use inline style instead because with StyleSheet.create we  cannot see which style is still used which is not

Reply:
StyleSheet.create also breaks type safety